spec/fixtures/prompt/prompt_template.yaml

```diff
@@ -0,0 +1,5 @@
+_type: prompt
+input_variables:
+  - adjective
+  - content
+template: Tell me a {adjective} joke about {content}.
```

Comment:
Does yaml support multi-line strings?

Reply:
That was my complaint with JSON -- that you can't do multiline strings and a lengthy template would just be a single line with tons of line break characters.